packages/regl-worldview/src/commands/FilledPolygons.js

```diff
@@ -46,16 +46,18 @@ function getEarcutPoints(points: Vec3[]): Vec3[] {
 
 type Props = {
   children: PolygonType[],
+  getHitmapId?: (PolygonType) => number,
   getHitmapProps: GetHitmapProps<PolygonType>,
   getObjectFromHitmapId: GetObjectFromHitmapId<PolygonType>,
 };
 
 // command to draw a filled polygon
 export default function FilledPolygons({
   children: polygons = [],
+  getHitmapId,
```

Comment:
I don't think this is necessary, `getHitmapId` would have already been passed through

Reply:
I added because I wanted to explicitly support the old api. Guess it's better to implicitly support it so we don't have to remove it in the future. @brianc  what do you think?

Reply:
yeah implicit probably the way to go - mostly just leaving it as a valid prop type was my concern so we don't break flow for folks who were using it.

Reply:
Makes sense to make flow work. I misunderstood you. Will add it back.